app/vue/package.json

```diff
@@ -26,13 +26,18 @@
     "prepare": "node ../../scripts/prepare.js"
   },
   "dependencies": {
+    "@storybook/addons": "5.2.0-beta.10",
     "@storybook/core": "5.2.0-beta.10",
     "common-tags": "^1.8.0",
     "core-js": "^3.0.1",
     "global": "^4.3.2",
-    "regenerator-runtime": "^0.12.1"
+    "regenerator-runtime": "^0.12.1",
+    "webpack": "^4.33.0"
```

Comment:
Why they now being part of the dependency specially to this package? And `webpack` are usually belongs to the devDep I believe... nope?

Reply:
app/vue is a devDependency for users, which has webpack as a normal dependency, because it's actually used in the runtime of the user.

most apps have this, that need something tweaked to the webpack config.